#Exception 

#ที่เรานำมายกตัวอย่างคือ การเกิดข้อผิดพลาดในการทำงานของโปรแกรมคำนวณหาผลหารของตัวเลข2จำนวน 
#ปัญหาของเราคือ ถ้าเราป้อน ตัวอักษรเข้าไปละ จะเกิดข้อผิดพลาดในการทำงาน
#ปัญหาถัดมา ถ้าป้อนตัวเลขที่2เป็น0 จะเกิดข้อผิดพลาดในการทำงาน

nember1=int(input("ป้อนตัวเลขที่1: "))
nember2=int(input("ป้อนตัวเลขที่2: "))

result= nember1/nember2
print("ผลลัพธ์ = ",result)

#เราสามารถใช้ try และ except ในการแก้ปัญหาข้อผิดพลาดได้  *********ใน program18.py